sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/Batcher.java

```diff
@@ -51,4 +51,7 @@
    * @return the list of metrics batched in this Batcher.
    */
   List<MetricData> completeCollectionCycle();
+
+  /** Does this batcher generate "delta" style metrics. The alternative is "cumulative". */
```

Comment:
```suggestion
  /** Returns whether this batcher generate "delta" style metrics. The alternative is "cumulative". */
```

Reply:
I'm going to accept your changes, then regret it when the formatter check fails.  ;)